index.html

```diff
@@ -3078,25 +3078,62 @@ <h3>Extensibility</h3>
+            Whenever possible, it is RECOMMENDED to re-use terms — and their corresponding URLs — defined by well-known, public vocabularies,
+            such as [[[?DCTERMS]]] [[?DCTERMS]] or [[[?schema-org]]] [[?schema-org]]. 
+            If that is not possible, developers MUST <em>define</em> a new URL for each term. 
+            When doing so, general [[LINKED-DATA]] principles must be followed, in particular:
+          </p>
+
           <ul>
-            <li>
-JSON-LD-based processors MUST produce an error when a JSON-LD context redefines
-any term in the
-<a href="https://www.w3.org/TR/json-ld/#dfn-active-context">active context</a>.
-The only way to change the definition of existing terms is to introduce a new
-term that clears the active context within the scope of that new term. Authors
-that are interested in this feature should read about the
-`@protected` feature in the JSON-LD 1.1 specification.
-            </li>
-          </ul>
+              <li>
+                Human-readable documentation MUST be published, describing the semantics of and
+                the constraints on the use of each term.
+              </li>
+              <li>
+                  It is RECOMMENDED to also publish the collection of all new terms as a machine-readable vocabulary 
+                  using [[[?RDF-SCHEMA]]] [[?RDF-SCHEMA]] (and, if necessary, OWL2 [[?OWL2-OVERVIEW]]).
+              </li>
+              <li>
+                  It MUST be possible to dereference the URL of a term, resulting in its description and/or formal definition.   
+              </li> 
+          </ul>       
+             
+          <p>
+            Developers SHOULD follow the <a data-cite="?LD-BP#vocabulary-checklist">detailed checklist</a> in
+            [[[?LD-BP]]] [[?LD-BP]] when defining a new vocabulary.
+          </p>
+          <p>
+              Furthermore, a machine-readable description (that is, a 
+              <a data-cite="JSON-LD11#dfn-context">JSON-LD Context document</a>) MUST be published at the URL specified
+              in the `@context` [=property=] for the vocabulary. 
+              This context MUST map each term to its corresponding URL, possibly accompanied by further
+              constraints like the type of the property value. A human-readable document describing the expected order of values for
+              the `@context` [=property=] is also expected to be published by any implementer seeking interoperability.     
+          </p>
+
+          <p class="note">
+              When processing the <a data-cite="JSON-LD11#dfn-active-context">active context</a> defined by the base JSON-LD Context
+              document <a href="#dfn-context">defined in this specification</a>, compliant JSON-LD-based processors produce an error when a JSON-LD context 
+              <em>redefines</em> any term.
```

Comment:
Hmm, this downgrades a previous normative requirement to just a note. I thought the group felt strongly about the previous normative statement (though, that's "just how modern JSON-LD Processors are expected to work if an `@protected` term is redefined". We should make sure the group is ok with this.

Reply:
What bothered me is that the previous normative sentence seemed to impose a normative requirement on the JSON-LD processor, ie, the way it handles `@protected`. Which is not for this specification to define, this is how JSON-LD 1.1 is defined. Hence this 'note' version which simply states how JSON LD processor behaves.

We may want to add something along the lines of 'per the defition of JSON-LD 1.1' with a reference to that spec. But it still is non-normative in _this_ spec.

Reply:
This is a misunderstanding, see my comment above. The VCDM is using `@protected` according to the JSON-LD 1.1 spec, there is no difference.